velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -379,5 +379,41 @@ TEST_F(DateTimeFunctionsTest, dayofWeekTs) {
   }
 }
 
+TEST_F(DateTimeFunctionsTest, dateDiffDate) {
+  const auto dateDiff = [&](std::optional<int32_t> endDate,
+                            std::optional<int32_t> startDate) {
+    return evaluateOnce<int32_t, int32_t>(
+        "date_diff(c0, c1)", {endDate, startDate}, {DATE(), DATE()});
+  };
+
+  const auto dateDiffAlias = [&](std::optional<int32_t> endDate,
+                                 std::optional<int32_t> startDate) {
+    return evaluateOnce<int32_t, int32_t>(
+        "datediff(c0, c1)", {endDate, startDate}, {DATE(), DATE()});
+  };
+  // Check null behaviors
```

Comment:
Remove these tests. They test the framework, not the function.

Reply:
Just removed it.